bin/fp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# This script is primarily for running Firefox Profiler on a saved profile.
```

Comment:
Also what do you think about naming this file `launch-fp.sh` so that it's more obvious about what it does?

Reply:
> Can you please add a few examples about how to use it in the comments here?

Ok 974ad6d7738e5f73ac29e390f9ee9c31e6414b99

> Also what do you think about naming this file `launch-fp.sh` so that it's more obvious about what it does?

No strong opinions on this and I'm symlinking to it anyway so ok 974ad6d7738e5f73ac29e390f9ee9c31e6414b99